apps/src/code-studio/components/SmallFooter.jsx

```diff
@@ -235,10 +220,8 @@ export default class SmallFooter extends React.Component {
           style={combinedBaseStyle}
           onClick={this.clickBase}
         >
-          <div
-            dangerouslySetInnerHTML={{
-              __html: decodeURIComponent(this.props.i18nDropdown)
-            }}
+          <UnsafeRenderedMarkdown
+            markdown={decodeURIComponent(this.props.i18nDropdown)}
```

Comment:
is it necessary to use `decodeURIComponent` still? I assume it was used here for things like the copyright symbol, but I also suspect that's handled by markdown

Reply:
Looks like we do: (removing decodeURIComponent from this particular line looked even worse)

![Screenshot 2019-04-22 at 1 30 06 PM](https://user-images.githubusercontent.com/46464143/56527657-d4709a00-6502-11e9-9938-623501f8ed07.png)

Reply:
We encode it here, not sure why: https://github.com/code-dot-org/code-dot-org/blob/180f7dc6d8b4917ed02a5006ddbd39613b2ee298/dashboard/app/views/layouts/_small_footer.html.haml#L7

Reply:
Strange